MMM-quote-of-the-day.js

```diff
@@ -36,7 +36,7 @@ Module.register('MMM-quote-of-the-day', {
         quoteTextDiv.className = "normal";
         quoteAuthorDiv.className = "small dimmed";
 
-        if (this.result.quoteText && this.result.quoteAuthor){
+        if (this.result.quoteText){
             quoteTextDiv.innerHTML = this.result.quoteText;
             quoteAuthorDiv.innerHTML = this.result.quoteAuthor;
```

Comment:
so maybe add a test here to add it only if quoteAuthor not null?

Reply:
quote can be provided without an author. in such a case, we must only display the quote and empty author

Reply:
But the line `quoteAuthorDiv.innerHTML = this.result.quoteAuthor;` will not fail here if quoteAuthor not in "result?
That's why I say it should be tested first

Reply:
> will not fail here if quoteAuthor not in "result?

no, it will no fail, author just will be empty

Reply:
Ok then